### DISCUSSION TITLE
Change in method used to determine Checkpoint Hash?

### DISCUSSION THREAD

**Discussion by mistermcluvin** [lllyasviel/stable-diffusion-webui-forge#1242] [Q&A]
Question. 
Was there a change on how Forge determines the Checkpoint hash?
I noticed when I post locally generated pictures on Civitai with the current version it can no longer associate it to the checkpoint? When I compared my current generation data to my older version of Forge I noticed the checkpoint hashes do not match? 

Example, Cyberrealistic V50 (SD1.5), same model file (shared folder).

Current Version of Forge:
cyberrealistic_v50
Description
Filename:	0.000_01\cyberrealistic_v50.safetensors
File size:	4GB
Hash:	**321b1198d8**
Modified:	2024-05-29 10:04

Older Version of Forge (version: [f0.0.17v1.8.0rc-latest-276-g29be1da7]).
cyberrealistic_v50
Description
Filename:	0.000_01\cyberrealistic_v50.safetensors
File size:	4GB
Hash:	**0b20a3f2cd**
Modified:	2024-05-29 10:04

It's really not a huge deal but I was just curious. 
Thanks!

**Comment by sunny-sko** [comment]
i replaced folder \cache from AUTOMATIC1111, and it help'd only in part,
for forge rehash some loras/chpnts anyway

**Reply by sunny-sko** [reply]
lil' correction:  forge writes 3 lists of hashes in file:


> Steps: 24, Sampler: IPNDM, Schedule type: Beta, CFG scale: 1, Distilled CFG Scale: 3.5, Seed: 3129685724, Size: 896x1152,
> 
> **Model hash:** **4610115bb0**, Model: flux1-dev, RNG: NV, Style Selector Enabled: True, 
> **Lora hashes:** " - Flux1 - vanta_black_V1.0: ea6991ec31ba, ImageUpgraderV2: e5598e0ef11d, Dever_Flux_Enhancer: 6be64a302443, FluxDFaeTasticDetails: 5be8e41a3a18, Simon_Stålenhag_Flux: 5497055a3e2d, ink_monsters: e3b0c44298fc",
> 
> Beta schedule alpha: 0.6, Beta schedule beta: 0.6, Version: f2.0.1v1.10.1-previous-442-g230e3911, Module 1: clip_l, Module 2: ae, Module 3: t5xxl_fp16,
> 
> **Hashes:** {"lora:Dever_Flux_Enhancer": "471ee7052a", "lora:FluxDFaeTasticDetails": "4259dc3287", "lora:ink_monsters": "a8079603fe", "model": "**4610115bb0**"}


and only third one is "correct" but civitai seems to reads from first and second, and not all of 'em are
